data/campaigns/Two_Brothers/scenarios/01_Rooting_Out_a_Mage.cfg

```diff
@@ -172,6 +172,11 @@ Will he heed the call? I do not know if he has kept the amulet; we have not spok
                 bonus=yes
                 carryover_percentage=40
             [/gold_carryover]
+#ifdef HARD
```

Comment:
I think you meant `#ifndef HARD`; though, I think `#ifdef EASY` might be more appropriate.